nixos/tests/postgresql.nix

```diff
@@ -0,0 +1,25 @@
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "postgresql";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ zagy ];
+  };
+
+  nodes = {
+    master =
+      { pkgs, config, ... }:
+
+      {
+        services.postgresql.enable = true;
+        services.postgresql.initialScript = "/etc/nixos/postgresql-init.sql";
```

Comment:
Oh well. Mainly because I don't know nix that much :)

Sounds good. Much easier indeed. Thanks – learnt something again.